From 8cfac9e2c77eb0f1608870bcc917341694eab805 Mon Sep 17 00:00:00 2001
From: Lars Op den Kamp <lars@opdenkamp.eu>
Date: Tue, 1 Dec 2015 14:58:03 +0100
Subject: [PATCH] changed: more stuff for renaming platform -> p8-platform

Conflicts:
	debian/control
	src/libcec/CMakeLists.txt
	src/libcec/SwigHelper.h
---
 debian/control                                                | 2 +-
 src/LibCecSharp/CecSharpTypes.h                               | 2 +-
 src/cec-client/CMakeLists.txt                                 | 6 +++---
 src/cec-client/cec-client.cpp                                 | 6 +++---
 src/cec-client/curses/CursesControl.cpp                       | 2 +-
 src/cec-client/env.h.in                                       | 2 +-
 src/cecc-client/CMakeLists.txt                                | 6 +++---
 src/cecc-client/env.h.in                                      | 2 +-
 src/libcec/CECClient.h                                        | 6 +++---
 src/libcec/CECInputBuffer.h                                   | 4 ++--
 src/libcec/CECProcessor.cpp                                   | 4 ++--
 src/libcec/CECProcessor.h                                     | 4 ++--
 src/libcec/CECTypeUtils.h                                     | 2 +-
 src/libcec/CMakeLists.txt                                     | 9 ++++-----
 src/libcec/LibCEC.cpp                                         | 4 ++--
 src/libcec/LibCEC.h                                           | 2 +-
 src/libcec/SwigHelper.h                                       | 7 ++++++-
 src/libcec/adapter/Pulse-Eight/USBCECAdapterCommands.cpp      | 2 +-
 src/libcec/adapter/Pulse-Eight/USBCECAdapterCommands.h        | 2 +-
 src/libcec/adapter/Pulse-Eight/USBCECAdapterCommunication.cpp | 4 ++--
 src/libcec/adapter/Pulse-Eight/USBCECAdapterCommunication.h   | 2 +-
 src/libcec/adapter/Pulse-Eight/USBCECAdapterDetection.cpp     | 2 +-
 src/libcec/adapter/Pulse-Eight/USBCECAdapterMessageQueue.cpp  | 2 +-
 src/libcec/adapter/Pulse-Eight/USBCECAdapterMessageQueue.h    | 6 +++---
 src/libcec/adapter/RPi/RPiCECAdapterCommunication.h           | 2 +-
 src/libcec/adapter/RPi/RPiCECAdapterMessageQueue.h            | 2 +-
 src/libcec/adapter/TDA995x/AdapterMessageQueue.h              | 2 +-
 src/libcec/adapter/TDA995x/TDA995xCECAdapterCommunication.cpp | 4 ++--
 src/libcec/adapter/TDA995x/TDA995xCECAdapterCommunication.h   | 6 +++---
 src/libcec/devices/CECBusDevice.cpp                           | 4 ++--
 src/libcec/devices/CECBusDevice.h                             | 2 +-
 src/libcec/env.h.in                                           | 2 +-
 src/libcec/implementations/AQCommandHandler.h                 | 2 +-
 src/libcec/implementations/CECCommandHandler.cpp              | 2 +-
 src/libcec/implementations/CECCommandHandler.h                | 2 +-
 src/libcec/implementations/PHCommandHandler.h                 | 2 +-
 src/libcec/implementations/RLCommandHandler.cpp               | 2 +-
 src/libcec/implementations/SLCommandHandler.cpp               | 2 +-
 src/libcec/platform/adl/adl-edid.cpp                          | 2 +-
 src/libcec/platform/drm/drm-edid.cpp                          | 2 +-
 src/libcec/platform/nvidia/nv-edid.cpp                        | 2 +-
 src/libcec/platform/posix/serialport.cpp                      | 2 +-
 src/libcec/platform/sockets/serialport.h                      | 4 ++--
 src/libcec/platform/windows/serialport.cpp                    | 2 +-
 44 files changed, 72 insertions(+), 68 deletions(-)

diff --git a/debian/control b/debian/control
index ee8690e..0487311 100644
--- a/debian/control
+++ b/debian/control
@@ -6,7 +6,7 @@ Build-Depends: debhelper (>= 7),
  pkg-config,
  libudev-dev,
  liblockdev1-dev,
- libplatform-dev,
+ libp8-platform-dev,
  python-dev,
  cmake (>= 2.8.9),
  swig,
diff --git a/src/LibCecSharp/CecSharpTypes.h b/src/LibCecSharp/CecSharpTypes.h
index eea75e3..4f26d54 100644
--- a/src/LibCecSharp/CecSharpTypes.h
+++ b/src/LibCecSharp/CecSharpTypes.h
@@ -31,7 +31,7 @@
 *     http://www.pulse-eight.net/
 */
 
-#include "platform/threads/mutex.h"
+#include <p8-platform/threads/mutex.h>
 #include <vcclr.h>
 #include <msclr/marshal.h>
 #include "../../include/cec.h"
diff --git a/src/cec-client/CMakeLists.txt b/src/cec-client/CMakeLists.txt
index 938b06e..2cb42bb 100644
--- a/src/cec-client/CMakeLists.txt
+++ b/src/cec-client/CMakeLists.txt
@@ -18,7 +18,7 @@ if (SUPPORTS_CXX11)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 endif()
 
-find_package(platform REQUIRED)
+find_package(p8-platform REQUIRED)
 find_package(Threads REQUIRED)
 
 set(cecclient_SOURCES cec-client.cpp)
@@ -31,7 +31,7 @@ endif()
 
 add_executable(cec-client ${cecclient_SOURCES})
 set_target_properties(cec-client PROPERTIES VERSION ${LIBCEC_VERSION_MAJOR}.${LIBCEC_VERSION_MINOR}.${LIBCEC_VERSION_PATCH})
-target_link_libraries(cec-client ${platform_LIBRARIES})
+target_link_libraries(cec-client ${p8-platform_LIBRARIES})
 target_link_libraries(cec-client ${CMAKE_THREAD_LIBS_INIT})
 
 if (NOT WIN32)
@@ -63,7 +63,7 @@ else()
   endif()
 endif()
 
-include_directories(${platform_INCLUDE_DIRS}
+include_directories(${p8-platform_INCLUDE_DIRS}
                     ${PROJECT_SOURCE_DIR}
                     ${PROJECT_SOURCE_DIR}/../../include)
 
diff --git a/src/cec-client/cec-client.cpp b/src/cec-client/cec-client.cpp
index 98d668c..4814149 100644
--- a/src/cec-client/cec-client.cpp
+++ b/src/cec-client/cec-client.cpp
@@ -42,9 +42,9 @@
 #include <sstream>
 #include <signal.h>
 #include <stdlib.h>
-#include "platform/os.h"
-#include "platform/util/StringUtils.h"
-#include "platform/threads/threads.h"
+#include <p8-platform/os.h>
+#include <p8-platform/util/StringUtils.h>
+#include <p8-platform/threads/threads.h>
 #if defined(HAVE_CURSES_API)
   #include "curses/CursesControl.h"
 #endif
diff --git a/src/cec-client/curses/CursesControl.cpp b/src/cec-client/curses/CursesControl.cpp
index 16752cc..c96ff4c 100644
--- a/src/cec-client/curses/CursesControl.cpp
+++ b/src/cec-client/curses/CursesControl.cpp
@@ -32,7 +32,7 @@
  */
 
 #include "CursesControl.h"
-#include "platform/util/StringUtils.h"
+#include <p8-platform/util/StringUtils.h>
 #include <curses.h>
 
 void CCursesControl::Init()
diff --git a/src/cec-client/env.h.in b/src/cec-client/env.h.in
index b7925aa..2f3fd71 100644
--- a/src/cec-client/env.h.in
+++ b/src/cec-client/env.h.in
@@ -34,7 +34,7 @@
  */
 
 #include "cectypes.h"
-#include "platform/os.h"
+#include <p8-platform/os.h>
 
 #ifdef UNUSED
 #elif defined(__GNUC__)
diff --git a/src/cecc-client/CMakeLists.txt b/src/cecc-client/CMakeLists.txt
index a0aacfb..3b8b9e8 100644
--- a/src/cecc-client/CMakeLists.txt
+++ b/src/cecc-client/CMakeLists.txt
@@ -12,14 +12,14 @@ include(CheckCSourceCompiles)
 include(CheckLibraryExists)
 include(CheckIncludeFiles)
 
-find_package(platform REQUIRED)
+find_package(p8-platform REQUIRED)
 find_package(Threads REQUIRED)
 
 set(ceccclient_SOURCES cecc-client.c)
 
 add_executable(cecc-client ${ceccclient_SOURCES})
 set_target_properties(cecc-client PROPERTIES VERSION ${LIBCEC_VERSION_MAJOR}.${LIBCEC_VERSION_MINOR}.${LIBCEC_VERSION_PATCH})
-target_link_libraries(cecc-client ${platform_LIBRARIES})
+target_link_libraries(cecc-client ${p8-platform_LIBRARIES})
 target_link_libraries(cecc-client ${CMAKE_THREAD_LIBS_INIT})
 
 if (NOT WIN32)
@@ -40,7 +40,7 @@ else()
   endif()
 endif()
 
-include_directories(${platform_INCLUDE_DIRS}
+include_directories(${p8-platform_INCLUDE_DIRS}
                     ${PROJECT_SOURCE_DIR}
                     ${PROJECT_SOURCE_DIR}/../../include)
 
diff --git a/src/cecc-client/env.h.in b/src/cecc-client/env.h.in
index 01f9d08..9b80c71 100644
--- a/src/cecc-client/env.h.in
+++ b/src/cecc-client/env.h.in
@@ -34,7 +34,7 @@
  */
 
 #include "cectypes.h"
-#include "platform/os.h"
+#include <p8-platform/os.h>
 
 #ifdef UNUSED
 #elif defined(__GNUC__)
diff --git a/src/libcec/CECClient.h b/src/libcec/CECClient.h
index 85a6428..3b86ff3 100644
--- a/src/libcec/CECClient.h
+++ b/src/libcec/CECClient.h
@@ -34,9 +34,9 @@
 
 #include "env.h"
 #include <string>
-#include "platform/threads/threads.h"
-#include "platform/util/buffer.h"
-#include "platform/threads/mutex.h"
+#include <p8-platform/threads/threads.h>
+#include <p8-platform/util/buffer.h>
+#include <p8-platform/threads/mutex.h>
 #include <memory>
 
 namespace CEC
diff --git a/src/libcec/CECInputBuffer.h b/src/libcec/CECInputBuffer.h
index 65f72ce..e38ab80 100644
--- a/src/libcec/CECInputBuffer.h
+++ b/src/libcec/CECInputBuffer.h
@@ -33,8 +33,8 @@
  */
 
 #include "env.h"
-#include "platform/threads/mutex.h"
-#include "platform/util/buffer.h"
+#include <p8-platform/threads/mutex.h>
+#include <p8-platform/util/buffer.h>
 
 namespace CEC
 {
diff --git a/src/libcec/CECProcessor.cpp b/src/libcec/CECProcessor.cpp
index 604b950..b5c607e 100644
--- a/src/libcec/CECProcessor.cpp
+++ b/src/libcec/CECProcessor.cpp
@@ -45,8 +45,8 @@
 #include "LibCEC.h"
 #include "CECClient.h"
 #include "CECTypeUtils.h"
-#include "platform/util/timeutils.h"
-#include "platform/util/util.h"
+#include <p8-platform/util/timeutils.h>
+#include <p8-platform/util/util.h>
 #include <stdio.h>
 
 using namespace CEC;
diff --git a/src/libcec/CECProcessor.h b/src/libcec/CECProcessor.h
index e022880..5ddd84b 100644
--- a/src/libcec/CECProcessor.h
+++ b/src/libcec/CECProcessor.h
@@ -35,8 +35,8 @@
 #include "env.h"
 #include <string>
 
-#include "platform/threads/threads.h"
-#include "platform/util/buffer.h"
+#include <p8-platform/threads/threads.h>
+#include <p8-platform/util/buffer.h>
 
 #include "adapter/AdapterCommunication.h"
 #include "devices/CECDeviceMap.h"
diff --git a/src/libcec/CECTypeUtils.h b/src/libcec/CECTypeUtils.h
index fd8046a..ecdad5f 100644
--- a/src/libcec/CECTypeUtils.h
+++ b/src/libcec/CECTypeUtils.h
@@ -33,7 +33,7 @@
  */
 
 #include "env.h"
-#include "platform/util/StringUtils.h"
+#include <p8-platform/util/StringUtils.h>
 
 namespace CEC
 {
diff --git a/src/libcec/CMakeLists.txt b/src/libcec/CMakeLists.txt
index 9696264..a494533 100644
--- a/src/libcec/CMakeLists.txt
+++ b/src/libcec/CMakeLists.txt
@@ -25,7 +25,7 @@ endif()
 
 find_package(Threads REQUIRED)
 
-include_directories(${platform_INCLUDE_DIRS}
+include_directories(${p8-platform_INCLUDE_DIRS}
                     ${PROJECT_SOURCE_DIR}
                     ${PROJECT_SOURCE_DIR}/../../include)
 
@@ -108,7 +108,6 @@ set(CEC_HEADERS devices/CECRecordingDevice.h
                 platform/adl/adl_defines.h
                 platform/adl/adl-edid.h
                 platform/adl/adl_sdk.h
-                platform/drm/drm-edid.cpp
                 platform/sockets/serialport.h
                 platform/X11/randr-edid.h
                 CECClient.h
@@ -146,7 +145,7 @@ list(APPEND CEC_SOURCES ${CEC_HEADERS}
                         ${CEC_SOURCES_PLATFORM})
 
 # write env.h
-set(LIBCEC_LIBREQUIRES "platform ${PLATFORM_LIBREQUIRES}")
+set(LIBCEC_LIBREQUIRES "p8-platform ${PLATFORM_LIBREQUIRES}")
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/env.h.in ${CMAKE_CURRENT_SOURCE_DIR}/env.h)
 if (WIN32)
   # write libcec.rc
@@ -178,8 +177,8 @@ add_library(cec SHARED ${CEC_SOURCES})
 install(TARGETS cec
         DESTINATION ${LIB_DESTINATION})
 set_target_properties(cec PROPERTIES VERSION   ${LIBCEC_VERSION_MAJOR}.${LIBCEC_VERSION_MINOR}.${LIBCEC_VERSION_PATCH}
-                                     SOVERSION ${LIBCEC_VERSION_MAJOR}.0)
-target_link_libraries(cec ${platform_LIBRARIES})
+                                     SOVERSION ${LIBCEC_VERSION_MAJOR})
+target_link_libraries(cec ${p8-platform_LIBRARIES})
 target_link_libraries(cec ${CMAKE_THREAD_LIBS_INIT})
 
 include(cmake/LinkPlatformSupport.cmake)
diff --git a/src/libcec/LibCEC.cpp b/src/libcec/LibCEC.cpp
index 5ccb8dd..528f3fe 100644
--- a/src/libcec/LibCEC.cpp
+++ b/src/libcec/LibCEC.cpp
@@ -41,8 +41,8 @@
 #include "devices/CECBusDevice.h"
 #include "devices/CECPlaybackDevice.h"
 #include "devices/CECTV.h"
-#include "platform/util/timeutils.h"
-#include "platform/util/util.h"
+#include <p8-platform/util/timeutils.h>
+#include <p8-platform/util/util.h>
 #include <stdio.h>
 #include <stdlib.h>
 
diff --git a/src/libcec/LibCEC.h b/src/libcec/LibCEC.h
index d9d1e7b..d1f6507 100644
--- a/src/libcec/LibCEC.h
+++ b/src/libcec/LibCEC.h
@@ -35,7 +35,7 @@
 #include "env.h"
 #include <string>
 #include "cec.h"
-#include "platform/util/buffer.h"
+#include <p8-platform/util/buffer.h>
 #include "CECTypeUtils.h"
 #include <memory>
 
diff --git a/src/libcec/SwigHelper.h b/src/libcec/SwigHelper.h
index 28a1b75..69101a9 100644
--- a/src/libcec/SwigHelper.h
+++ b/src/libcec/SwigHelper.h
@@ -44,7 +44,12 @@
 #include "cectypes.h"
 #include "cec.h"
 #include "CECTypeUtils.h"
-#include "platform/threads/mutex.h"
+#include <p8-platform/threads/mutex.h>
+/** XXX only to keep the IDE happy, using the actual Python.h with the correct system version when building */
+#ifndef Py_PYTHON_H
+#include <python2.7/Python.h>
+#include <assert.h>
+#endif
 
 /** XXX python does a #define PLATFORM, which causes a collision with our PLATFORM namespace */
 #ifdef _platform_tmp
diff --git a/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommands.cpp b/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommands.cpp
index 0ef6f78..cae20d4 100644
--- a/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommands.cpp
+++ b/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommands.cpp
@@ -39,7 +39,7 @@
 #include "LibCEC.h"
 #include "CECProcessor.h"
 #include "CECTypeUtils.h"
-#include "platform/util/util.h"
+#include <p8-platform/util/util.h>
 #include <stdio.h>
 
 using namespace CEC;
diff --git a/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommands.h b/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommands.h
index ece1bc0..305e815 100644
--- a/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommands.h
+++ b/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommands.h
@@ -33,7 +33,7 @@
  */
 
 #include "env.h"
-#include "platform/threads/mutex.h"
+#include <p8-platform/threads/mutex.h>
 #include "USBCECAdapterMessage.h"
 
 namespace CEC
diff --git a/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommunication.cpp b/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommunication.cpp
index 3c41963..fa35cd3 100644
--- a/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommunication.cpp
+++ b/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommunication.cpp
@@ -39,8 +39,8 @@
 #include "USBCECAdapterMessage.h"
 #include "USBCECAdapterDetection.h"
 #include "platform/sockets/serialport.h"
-#include "platform/util/timeutils.h"
-#include "platform/util/util.h"
+#include <p8-platform/util/timeutils.h>
+#include <p8-platform/util/util.h>
 #include "platform/util/edid.h"
 #include "platform/adl/adl-edid.h"
 #include "platform/nvidia/nv-edid.h"
diff --git a/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommunication.h b/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommunication.h
index 3bb8967..3119be5 100644
--- a/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommunication.h
+++ b/src/libcec/adapter/Pulse-Eight/USBCECAdapterCommunication.h
@@ -33,7 +33,7 @@
  */
 
 #include "env.h"
-#include "platform/threads/threads.h"
+#include <p8-platform/threads/threads.h>
 #include "adapter/AdapterCommunication.h"
 #include "USBCECAdapterMessage.h"
 
diff --git a/src/libcec/adapter/Pulse-Eight/USBCECAdapterDetection.cpp b/src/libcec/adapter/Pulse-Eight/USBCECAdapterDetection.cpp
index 391eada..f46e4cf 100644
--- a/src/libcec/adapter/Pulse-Eight/USBCECAdapterDetection.cpp
+++ b/src/libcec/adapter/Pulse-Eight/USBCECAdapterDetection.cpp
@@ -71,7 +71,7 @@ extern "C" {
 #include <string>
 #include <algorithm>
 #include <stdio.h>
-#include "platform/util/StringUtils.h"
+#include <p8-platform/util/StringUtils.h>
 
 #define CEC_VID  0x2548
 #define CEC_PID  0x1001
diff --git a/src/libcec/adapter/Pulse-Eight/USBCECAdapterMessageQueue.cpp b/src/libcec/adapter/Pulse-Eight/USBCECAdapterMessageQueue.cpp
index e86da50..0afccd7 100644
--- a/src/libcec/adapter/Pulse-Eight/USBCECAdapterMessageQueue.cpp
+++ b/src/libcec/adapter/Pulse-Eight/USBCECAdapterMessageQueue.cpp
@@ -36,7 +36,7 @@
 
 #include "USBCECAdapterCommunication.h"
 #include "USBCECAdapterMessage.h"
-#include "platform/sockets/socket.h"
+#include <p8-platform/sockets/socket.h>
 #include "LibCEC.h"
 
 using namespace CEC;
diff --git a/src/libcec/adapter/Pulse-Eight/USBCECAdapterMessageQueue.h b/src/libcec/adapter/Pulse-Eight/USBCECAdapterMessageQueue.h
index c04fd66..25586d6 100644
--- a/src/libcec/adapter/Pulse-Eight/USBCECAdapterMessageQueue.h
+++ b/src/libcec/adapter/Pulse-Eight/USBCECAdapterMessageQueue.h
@@ -33,9 +33,9 @@
  */
 
 #include "env.h"
-#include "platform/threads/threads.h"
-#include "platform/util/buffer.h"
-#include "platform/util/timeutils.h"
+#include <p8-platform/threads/threads.h>
+#include <p8-platform/util/buffer.h>
+#include <p8-platform/util/timeutils.h>
 #include <map>
 #include "USBCECAdapterMessage.h"
 
diff --git a/src/libcec/adapter/RPi/RPiCECAdapterCommunication.h b/src/libcec/adapter/RPi/RPiCECAdapterCommunication.h
index 6024a27..1e175f6 100644
--- a/src/libcec/adapter/RPi/RPiCECAdapterCommunication.h
+++ b/src/libcec/adapter/RPi/RPiCECAdapterCommunication.h
@@ -36,7 +36,7 @@
 #if defined(HAVE_RPI_API)
 
 #include "adapter/AdapterCommunication.h"
-#include "platform/threads/threads.h"
+#include <p8-platform/threads/threads.h>
 
 #define RPI_ADAPTER_VID 0x2708
 #define RPI_ADAPTER_PID 0x1001
diff --git a/src/libcec/adapter/RPi/RPiCECAdapterMessageQueue.h b/src/libcec/adapter/RPi/RPiCECAdapterMessageQueue.h
index bd2b9c9..3ac1333 100644
--- a/src/libcec/adapter/RPi/RPiCECAdapterMessageQueue.h
+++ b/src/libcec/adapter/RPi/RPiCECAdapterMessageQueue.h
@@ -33,7 +33,7 @@
  */
 
 #include "env.h"
-#include "platform/util/buffer.h"
+#include <p8-platform/util/buffer.h>
 #include <map>
 #include "adapter/AdapterCommunication.h"
 
diff --git a/src/libcec/adapter/TDA995x/AdapterMessageQueue.h b/src/libcec/adapter/TDA995x/AdapterMessageQueue.h
index 28160bd..52aadea 100644
--- a/src/libcec/adapter/TDA995x/AdapterMessageQueue.h
+++ b/src/libcec/adapter/TDA995x/AdapterMessageQueue.h
@@ -33,7 +33,7 @@
  */
 
 #include "env.h"
-#include "platform/threads/mutex.h"
+#include <p8-platform/threads/mutex.h>
 
 namespace CEC
 {
diff --git a/src/libcec/adapter/TDA995x/TDA995xCECAdapterCommunication.cpp b/src/libcec/adapter/TDA995x/TDA995xCECAdapterCommunication.cpp
index d008afa..db288c2 100644
--- a/src/libcec/adapter/TDA995x/TDA995xCECAdapterCommunication.cpp
+++ b/src/libcec/adapter/TDA995x/TDA995xCECAdapterCommunication.cpp
@@ -38,8 +38,8 @@
 
 #include "CECTypeUtils.h"
 #include "LibCEC.h"
-#include "platform/sockets/cdevsocket.h"
-#include "platform/util/buffer.h"
+#include <p8-platform/sockets/cdevsocket.h>
+#include <p8-platform/util/buffer.h>
 
 extern "C" {
 #define __cec_h__
diff --git a/src/libcec/adapter/TDA995x/TDA995xCECAdapterCommunication.h b/src/libcec/adapter/TDA995x/TDA995xCECAdapterCommunication.h
index f716a4c..3aa6320 100644
--- a/src/libcec/adapter/TDA995x/TDA995xCECAdapterCommunication.h
+++ b/src/libcec/adapter/TDA995x/TDA995xCECAdapterCommunication.h
@@ -35,9 +35,9 @@
 #include "env.h"
 #if defined(HAVE_TDA995X_API)
 
-#include "platform/threads/mutex.h"
-#include "platform/threads/threads.h"
-#include "platform/sockets/socket.h"
+#include <p8-platform/threads/mutex.h>
+#include <p8-platform/threads/threads.h>
+#include <p8-platform/sockets/socket.h>
 #include "adapter/AdapterCommunication.h"
 #include <map>
 
diff --git a/src/libcec/devices/CECBusDevice.cpp b/src/libcec/devices/CECBusDevice.cpp
index 26d585f..3907e47 100644
--- a/src/libcec/devices/CECBusDevice.cpp
+++ b/src/libcec/devices/CECBusDevice.cpp
@@ -46,8 +46,8 @@
 #include "implementations/AQCommandHandler.h"
 #include "LibCEC.h"
 #include "CECTypeUtils.h"
-#include "platform/util/timeutils.h"
-#include "platform/util/util.h"
+#include <p8-platform/util/timeutils.h>
+#include <p8-platform/util/util.h>
 
 #include "CECAudioSystem.h"
 #include "CECPlaybackDevice.h"
diff --git a/src/libcec/devices/CECBusDevice.h b/src/libcec/devices/CECBusDevice.h
index 35b286c..1d55e9e 100644
--- a/src/libcec/devices/CECBusDevice.h
+++ b/src/libcec/devices/CECBusDevice.h
@@ -36,7 +36,7 @@
 #include <set>
 #include <map>
 #include <string>
-#include "platform/threads/mutex.h"
+#include <p8-platform/threads/mutex.h>
 #include <memory>
 
 namespace CEC
diff --git a/src/libcec/env.h.in b/src/libcec/env.h.in
index ff20e49..f26a519 100644
--- a/src/libcec/env.h.in
+++ b/src/libcec/env.h.in
@@ -34,7 +34,7 @@
  */
 
 #include "cectypes.h"
-#include "platform/os.h"
+#include <p8-platform/os.h>
 
 #ifdef UNUSED
 #elif defined(__GNUC__)
diff --git a/src/libcec/implementations/AQCommandHandler.h b/src/libcec/implementations/AQCommandHandler.h
index 344fb26..4e2d43f 100644
--- a/src/libcec/implementations/AQCommandHandler.h
+++ b/src/libcec/implementations/AQCommandHandler.h
@@ -34,7 +34,7 @@
 
 #include "env.h"
 #include "CECCommandHandler.h"
-#include "platform/threads/threads.h"
+#include <p8-platform/threads/threads.h>
 
 namespace CEC
 {
diff --git a/src/libcec/implementations/CECCommandHandler.cpp b/src/libcec/implementations/CECCommandHandler.cpp
index e898cdf..b4b9819 100644
--- a/src/libcec/implementations/CECCommandHandler.cpp
+++ b/src/libcec/implementations/CECCommandHandler.cpp
@@ -41,7 +41,7 @@
 #include "CECProcessor.h"
 #include "LibCEC.h"
 #include "CECTypeUtils.h"
-#include "platform/util/util.h"
+#include <p8-platform/util/util.h>
 
 using namespace CEC;
 using namespace PLATFORM;
diff --git a/src/libcec/implementations/CECCommandHandler.h b/src/libcec/implementations/CECCommandHandler.h
index cb72653..c60078a 100644
--- a/src/libcec/implementations/CECCommandHandler.h
+++ b/src/libcec/implementations/CECCommandHandler.h
@@ -36,7 +36,7 @@
 #include <vector>
 #include <string>
 #include <map>
-#include "platform/threads/mutex.h"
+#include <p8-platform/threads/mutex.h>
 
 namespace CEC
 {
diff --git a/src/libcec/implementations/PHCommandHandler.h b/src/libcec/implementations/PHCommandHandler.h
index 4497fc0..af42f12 100644
--- a/src/libcec/implementations/PHCommandHandler.h
+++ b/src/libcec/implementations/PHCommandHandler.h
@@ -34,7 +34,7 @@
 
 #include "env.h"
 #include "CECCommandHandler.h"
-#include "platform/threads/threads.h"
+#include <p8-platform/threads/threads.h>
 
 namespace CEC
 {
diff --git a/src/libcec/implementations/RLCommandHandler.cpp b/src/libcec/implementations/RLCommandHandler.cpp
index fb5f6be..aa0cca8 100644
--- a/src/libcec/implementations/RLCommandHandler.cpp
+++ b/src/libcec/implementations/RLCommandHandler.cpp
@@ -34,7 +34,7 @@
 #include "env.h"
 #include "RLCommandHandler.h"
 
-#include "platform/util/timeutils.h"
+#include <p8-platform/util/timeutils.h>
 #include "devices/CECBusDevice.h"
 #include "CECProcessor.h"
 #include "LibCEC.h"
diff --git a/src/libcec/implementations/SLCommandHandler.cpp b/src/libcec/implementations/SLCommandHandler.cpp
index 2b3c2dd..80262c2 100644
--- a/src/libcec/implementations/SLCommandHandler.cpp
+++ b/src/libcec/implementations/SLCommandHandler.cpp
@@ -34,7 +34,7 @@
 #include "env.h"
 #include "SLCommandHandler.h"
 
-#include "platform/util/timeutils.h"
+#include <p8-platform/util/timeutils.h>
 #include "devices/CECBusDevice.h"
 #include "devices/CECPlaybackDevice.h"
 #include "CECProcessor.h"
diff --git a/src/libcec/platform/adl/adl-edid.cpp b/src/libcec/platform/adl/adl-edid.cpp
index 9b44187..4617107 100644
--- a/src/libcec/platform/adl/adl-edid.cpp
+++ b/src/libcec/platform/adl/adl-edid.cpp
@@ -36,7 +36,7 @@
 
 // for dlsym and friends
 #if defined(__WINDOWS__)
-#include "platform/windows/dlfcn-win32.h"
+#include <p8-platform/windows/dlfcn-win32.h>
 #endif
 
 using namespace PLATFORM;
diff --git a/src/libcec/platform/drm/drm-edid.cpp b/src/libcec/platform/drm/drm-edid.cpp
index 279282b..a5512f1 100644
--- a/src/libcec/platform/drm/drm-edid.cpp
+++ b/src/libcec/platform/drm/drm-edid.cpp
@@ -33,7 +33,7 @@
 #include "env.h"
 #ifdef HAS_DRM_EDID_PARSER
 
-#include "platform/os.h"
+#include <p8-platform/os.h>
 #include "drm-edid.h"
 #include <dirent.h>
 #include <fstream>
diff --git a/src/libcec/platform/nvidia/nv-edid.cpp b/src/libcec/platform/nvidia/nv-edid.cpp
index 0a0575c..fa4d7d6 100644
--- a/src/libcec/platform/nvidia/nv-edid.cpp
+++ b/src/libcec/platform/nvidia/nv-edid.cpp
@@ -33,7 +33,7 @@
 
 #include "env.h"
 
-#include "platform/os.h"
+#include <p8-platform/os.h>
 #include "nv-edid.h"
 #include <stdio.h>
 
diff --git a/src/libcec/platform/posix/serialport.cpp b/src/libcec/platform/posix/serialport.cpp
index 3fb1e7d..b14c202 100644
--- a/src/libcec/platform/posix/serialport.cpp
+++ b/src/libcec/platform/posix/serialport.cpp
@@ -36,7 +36,7 @@
 #include <fcntl.h>
 #include "../sockets/serialport.h"
 #include "../util/baudrate.h"
-#include "platform/posix/os-socket.h"
+#include <p8-platform/posix/os-socket.h>
 
 #if defined(__APPLE__) || defined(__FreeBSD__)
 #ifndef XCASE
diff --git a/src/libcec/platform/sockets/serialport.h b/src/libcec/platform/sockets/serialport.h
index abd6bdb..127f0cc 100644
--- a/src/libcec/platform/sockets/serialport.h
+++ b/src/libcec/platform/sockets/serialport.h
@@ -33,7 +33,7 @@
  */
 
 #include "env.h"
-#include "platform/util/buffer.h"
+#include <p8-platform/util/buffer.h>
 
 #include <string>
 #include <stdint.h>
@@ -42,7 +42,7 @@
 #include <termios.h>
 #endif
 
-#include "platform/sockets/socket.h"
+#include <p8-platform/sockets/socket.h>
 
 namespace PLATFORM
 {
diff --git a/src/libcec/platform/windows/serialport.cpp b/src/libcec/platform/windows/serialport.cpp
index 834c186..c1ef0a5 100644
--- a/src/libcec/platform/windows/serialport.cpp
+++ b/src/libcec/platform/windows/serialport.cpp
@@ -33,7 +33,7 @@
 
 #include "../sockets/serialport.h"
 #include "../util/baudrate.h"
-#include "platform/util/timeutils.h"
+#include <p8-platform/util/timeutils.h>
 
 using namespace PLATFORM;
 
-- 
2.1.0

